src/transmission_electron.py

```diff
@@ -0,0 +1,274 @@
+                    List containing the optic parameters
+
+        """
+        return None
+
+    @staticmethod
+    def get_image_data(file_paths, sim_path):
+        """Run simulator and return data.
+
+        Parameters
+        ----------
+        sim_path : str
+            relative path to local TEM simulator installation
+        file_paths : dict of type str to str
+             Dict of file paths that includes keys:
+                inp_file
+                    relative path to populated .inp file
+                mrc_file
+                    relative path to ouput mrc file
+                    sim_params : dict
+        Returns
+        -------
+        List containing parsed .mrc data from Simulator
+        """
+        # SIMULATOR_BIN = Path(sim_path)  # might have to change depending on OS
+        # inp_file = Path(file_paths["inp_file"])
+
+        # cmd = "{0} {1}".format(SIMULATOR_BIN, inp_file)
+        # os.system(shlex.quote(cmd))
+
+        return []
+
+    @staticmethod
+    def generate_parameters_dictionary(file_paths, sim_params):
+        """Compile experiment data into .inp friendly file for use in TEM-simulator.
+
+        Parameters
+        ----------
+        file_paths : dict of type str to str
+             Dict of file paths that includes keys:
+                pdb_file
+                    Relative path to pdb input file
+                crd_file
+                    Relative path to populated crd file
+                mrc_file
+                    Relative path to desired output mrc file
+                log_file
+                    Relative path to desired output log file
+        sim_params : dict
+            Dict of simulation parameters that includes keys:
+                seed : str maps to int
+                particle_mrcout : str maps to bool
+                sample_dimensions : str maps to list
+                beam_params : str maps to list
+                detector_params : str maps to list
+                optic_params : str maps to list
+
+        Returns
+        -------
+        param_dictionary : dict
+            .inp friendly dictionary containing simulation input parameters.
+        """
+        param_dictionary = {}
+        return param_dictionary
+
+    @staticmethod
+    def write_inp_file(param_dict, file_paths):
+        """Write simulation parameters to .inp file for use by the TEM-simulator.
+
+        Parameters
+        ----------
+        param_dict : dict
+            .inp friendly dictionary containing simulation input parameters.
+        file_paths : dict of type str to str
+            Dict of file paths that includes keys:
+                inp_file
+                    Relative path to input file to be populated with parameters
+        """
+        return None
+
+    @staticmethod
+    def extract_particles(micrograph, sim_param_arrays, file_paths, pad):
+        """Format and write molecular model data to crd_file for use in TEM-simulator.
```

Comment:
This docstring is not correct (this is the same docstring as another function above)